documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces-payload.adoc

```diff
@@ -132,41 +132,55 @@ h|*Interface* h|*interface-shortName*
 |Asset Administration Shell Basic Discovery Interface |AAS-DISCOVERY
 |===
 
-The value for the interface attribute is “\{interface-shortName}-\{interface-version}”.
+The value for the interface attribute shall be “\{interface-shortName}-\{interface-version}”.
 
-The interface-version of this specification is “3.0”, e.g. the entry for the Asset Administration Shell Interface is “AAS-3.0”.
+The interface-version of this specification is “3.1”, e.g. the entry for the Asset Administration Shell Interface is “AAS-3.1”.
 
-See the following example for a descriptor with several endpoints:
+See the following example for descriptor endpoints:
 
 [source,json,linenums]
 ----
-{
   "endpoints": [
     {
       "protocolInformation": {
         "endpointAddress": "https://localhost:1234",
         "endpointProtocolVersion": "1.1"
       },
-      "interface": "AAS-3.0"
+      "interface": "SUBMODEL-3.1"
     },
-    {
+	{
       "protocolInformation": {
-        "endpointAddress": "opc.tcp://localhost:4840"
-      },
-      "interface": "AAS-3.0"
-    },
-    {
-      "protocolInformation": {
-        "endpointAddress": "https://localhost:5678",
-        "endpointProtocolVersion": "1.1",
-        "subprotocol": "OPC UA Basic SOAP",
-        "subprotocolBody": "ns=2;s=MyAAS",
-        "subprotocolBodyEncoding": "application/soap+xml"
+        "endpointAddress": "https://localhost:1234",
```

Comment:
This example is invalid since both entries point to the very same endpoint `https://localhost:1234` while specifying a different interface version. As the prefix `/api/v{major}.{minor}` is now optional, this would cause a conflict.
To resolve this, the `endpointAddress` should be different for the entries, e.g., by changing the port or by adding a suffix like `v3.1`.

Reply:
yes, seems incorrect (I only changed the id-attribute but not the address: probably it was not ok in previous version as well. 

Although in https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2024/10/IDTA-01002-3-0-3_SpecificationAssetAdministrationShell_Part2_API.pdf the sentence "for a descriptor with several endpoints:" is still contained, the examples are not..."

Conformant to https://admin-shell-io.github.io/aas-specs-antora/IDTA-01002/v3.1/http-rest-api/http-rest-api.html#api-versioning I would now suggest

"endpointAddress": "https://localhost:1234/api/3/submodel"